edx_event_bus_kafka/consumer/apps.py

```diff
@@ -0,0 +1,28 @@
+"""App for consuming Kafka events. Comprises a management command for listening to a topic and supporting methods."""
+from django.apps import AppConfig
+from edx_django_utils.plugins.constants import PluginSettings
+
+
+class ConsumerApp(AppConfig):
```

Comment:
I think we can remove this whole file -- there's already an AppConfig in `edx_event_bus_kafka.apps` and we might not need one at all (this will probably just be a regular library) and the rest is plugin code. Probably want to remove `admin.py` as well.

Reply:
(Also models.py. Probably want another PR to de-appify the existing code too...)